ejemplo.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>No lo se aun</title>
+	<link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+	<section>
+		<figure class="ejemplo">
```

Comment:
although using a figure tag is ok in this case you could avoid this wrapper element and make the section be the father of the images and div